dali/server/daldap.cpp

```diff
@@ -196,7 +197,7 @@ class CDaliLdapConnection: implements IDaliLdapConnection, public CInterface
                 ERRLOG("LDAP: getPermissions(%s) scope=%s user=%s digital signature support not available",key?key:"NULL",obj?obj:"NULL",username.str());
         }
 
-        if (!isEmptyString(user->credentials().getPassword()))
+        if (!isEmptyString(user->credentials().getPassword()) && !isWorkunitDAToken(user->credentials().getPassword()))
```

Comment:
You verify the password only if it is not empty and it is not a WUDAToken, which is correct. But, what if the password is empty and the authentication is required?

Reply:
We discussed this at length with RKC, Jake, Tony and got Flavio's approval that a password will no longer be required in this call.  But if one is provided, it should be checked